generator-plugin/src/test/java/uk/gov/justice/maven/generator/io/files/parser/generator/ProjectDependencyLoaderTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.justice.maven.generator.io.files.parser.generator;
+
+import static org.hamcrest.CoreMatchers.endsWith;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.net.MalformedURLException;
+import java.net.URLClassLoader;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import org.apache.maven.artifact.DependencyResolutionRequiredException;
+import org.apache.maven.project.MavenProject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ProjectDependencyLoaderTest {
+
+    @Mock
+    private MavenProject project;
+
+    @InjectMocks
+    private ProjectDependencyLoader projectDependencyLoader;
+
+    @Test
+    public void loadProjectDependenciesTest() throws DependencyResolutionRequiredException, MalformedURLException {
```

Comment:
ideally the tests should start with the word "should" then a description of what is being tested